app/build.gradle

```diff
@@ -235,3 +235,9 @@ lintKotlinTest {
 sentry {
     ignoredBuildTypes = ["debug"]
 }
+
+tasks.withType(org.jetbrains.kotlin.gradle.tasks.KotlinCompile).configureEach {
+    kotlinOptions {
+        jvmTarget = '1.8'
```

Comment:
The ksp task does not pick up the jvmTarget from android.kotlinOptions, and this now results in an error with Gradle 8.0